frontend/pages/ManageControlsPage/MacOSSetup/_styles.scss

```diff
@@ -0,0 +1,26 @@
+.macos-setup {
+  > p {
+    font-size: $x-small;
+    margin: $pad-xxlarge 0;
+  }
+
+  &__empty-state {
```

Comment:
Maybe we could re-use EmptyTable.tsx (possibly renaming it EmptyState.tsx) so we don't have to restyle empty state across the app.